<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Materials and Methods | Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity</title>
  <meta name="description" content="This is the supplementary material accompanying the paper “Division of Labor Promotes the Entrenchment of Multicellularity”." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Materials and Methods | Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is the supplementary material accompanying the paper “Division of Labor Promotes the Entrenchment of Multicellularity”." />
  <meta name="github-repo" content="kgskocelas/mt2" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Materials and Methods | Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity" />
  
  <meta name="twitter:description" content="This is the supplementary material accompanying the paper “Division of Labor Promotes the Entrenchment of Multicellularity”." />
  

<meta name="author" content="Peter L. Conlin, Heather J. Goldsby, Eric Libby, Katherine G. Skocelas, William C. Ratcliff, Charles Ofria, Benjamin Kerr" />


<meta name="date" content="2023-04-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="distributed-dirt-model.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity</a></li>

<li class="divider"></li>
<li><a href="index.html#introduction" id="toc-introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="materials-and-methods.html#materials-and-methods" id="toc-materials-and-methods"><span class="toc-section-number">2</span> Materials and Methods</a>
<ul>
<li><a href="materials-and-methods.html#avida" id="toc-avida"><span class="toc-section-number">2.1</span> Avida</a>
<ul>
<li><a href="materials-and-methods.html#time" id="toc-time"><span class="toc-section-number">2.1.1</span> Time</a></li>
<li><a href="materials-and-methods.html#instructions-and-functions" id="toc-instructions-and-functions"><span class="toc-section-number">2.1.2</span> Instructions and functions</a></li>
<li><a href="materials-and-methods.html#workload" id="toc-workload"><span class="toc-section-number">2.1.3</span> Workload</a></li>
<li><a href="materials-and-methods.html#mutations" id="toc-mutations"><span class="toc-section-number">2.1.4</span> Mutations</a></li>
<li><a href="materials-and-methods.html#baseline-evolution-experiment" id="toc-baseline-evolution-experiment"><span class="toc-section-number">2.1.5</span> Baseline evolution experiment</a></li>
</ul></li>
<li><a href="materials-and-methods.html#evolution-of-multicellularity" id="toc-evolution-of-multicellularity"><span class="toc-section-number">2.2</span> Evolution of multicellularity</a></li>
<li><a href="materials-and-methods.html#cost-of-multicellularity" id="toc-cost-of-multicellularity"><span class="toc-section-number">2.3</span> Cost of multicellularity</a></li>
<li><a href="materials-and-methods.html#entrenchment" id="toc-entrenchment"><span class="toc-section-number">2.4</span> Entrenchment</a></li>
<li><a href="materials-and-methods.html#generation-and-assessment-of-unicellular-revertants" id="toc-generation-and-assessment-of-unicellular-revertants"><span class="toc-section-number">2.5</span> Generation and assessment of unicellular revertants</a>
<ul>
<li><a href="materials-and-methods.html#growth-assay-and-measuring-relative-fitness" id="toc-growth-assay-and-measuring-relative-fitness"><span class="toc-section-number">2.5.1</span> Growth assay and measuring relative fitness</a></li>
<li><a href="materials-and-methods.html#identification-of-unicellular-revertants" id="toc-identification-of-unicellular-revertants"><span class="toc-section-number">2.5.2</span> Identification of unicellular revertants</a></li>
</ul></li>
<li><a href="materials-and-methods.html#distribution-of-fitness-effects-of-unicellular-reversion-mutations" id="toc-distribution-of-fitness-effects-of-unicellular-reversion-mutations"><span class="toc-section-number">2.6</span> Distribution of fitness effects of unicellular reversion mutations</a></li>
<li><a href="materials-and-methods.html#reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations" id="toc-reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations"><span class="toc-section-number">2.7</span> Reversion mutation identities and the determinants of the number of viable reversion mutations</a></li>
<li><a href="materials-and-methods.html#fidelity" id="toc-fidelity"><span class="toc-section-number">2.8</span> Fidelity</a></li>
<li><a href="materials-and-methods.html#effect-of-time-spent-evolving-as-a-multicell" id="toc-effect-of-time-spent-evolving-as-a-multicell"><span class="toc-section-number">2.9</span> Effect of time spent evolving as a multicell</a></li>
</ul></li>
<li><a href="distributed-dirt-model.html#distributed-dirt-model" id="toc-distributed-dirt-model"><span class="toc-section-number">3</span> Distributed Dirt Model</a></li>
<li><a href="references.html#references" id="toc-references">References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="materials-and-methods" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> Materials and Methods</h1>
<div id="avida" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Avida</h2>
<p>Digital evolution experiments leverage the power of computers to ensure rapid generations (millions within days), perfect data collection (evolutionary lineages stored at fine temporal resolution), and high levels of experimental control (selective pressures toggled with configuration files). Here we use the software platform Avida <span class="citation">(<a href="#ref-ofria2004avida" role="doc-biblioref">Ofria and Wilke 2004</a>)</span> which has previously been employed to study the evolutionary origin of complex features <span class="citation">(<a href="#ref-Lenski2003" role="doc-biblioref">Lenski et al. 2003</a>)</span>, division of labor <span class="citation">(<a href="#ref-goldsby2012task" role="doc-biblioref">Goldsby et al. 2012</a>, <a href="#ref-goldsby2014evolutionary" role="doc-biblioref">2014</a>)</span>, and adaptive radiation <span class="citation">(<a href="#ref-Chow2004" role="doc-biblioref">Chow et al. 2004</a>)</span>. In this study, we focus on the entrenchment of higher-level units formed via fraternal transitions, where a higher-level unit (a digital “multicell”) is composed of a set of related lower-level units (digital “cells”). We maintain a population of 1,000 organisms, where each organism can be either unicellular or multicellular. Each cell consists of a program (i.e., its genome), where the instructions encode metabolism, development and reproduction.</p>
<div id="time" class="section level3" number="2.1.1">
<h3><span class="header-section-number">2.1.1</span> Time</h3>
<p>The standard unit of time in Avida is an “update.” On average, cells receive 30 CPU cycles every update. The other unit of time in this study is a “generation.” Along a line of descent, every time an offspring is produced the generation value increments by one. Within an evolving population, generation is calculated at the level of cells in the following way. Every cell gets the generation value of its parent cell incremented by one. Time in terms of generations is simply the mean generation of all cells within the population at any instant. Note that one unit of generation time within a population will track the average population turnover time of unicellular organisms, but will only be a fraction of the turnover time for multicellular organisms.</p>
</div>
<div id="instructions-and-functions" class="section level3" number="2.1.2">
<h3><span class="header-section-number">2.1.2</span> Instructions and functions</h3>
<p>The Avidian genome is composed of instructions that enable cells to acquire resources, communicate with other cells in a multicellular context, reproduce within the body, and determine their eligibility to produce a propagule to initiate a new organism. A complete list of the instructions used in our evolution experiments, separated by category, are provided in Tables <a href="materials-and-methods.html#fig:Avida-instructions-No-operation-Table">2.1</a>, <a href="materials-and-methods.html#fig:Avida-instructions-Hardware-control-Table">2.2</a>,
<a href="materials-and-methods.html#fig:Avida-instructions-Math-Table">2.3</a>,
<a href="materials-and-methods.html#fig:Avida-instructions-Interaction-Table">2.4</a>, and
<a href="materials-and-methods.html#fig:Avida-instructions-Biological-Table">2.5</a>.</p>
<div class="figure"><span style="display:block;" id="fig:Avida-instructions-No-operation-Table"></span>
<img src="images/Avida_instructions_No-operation_Table.png" alt="**No-operation instructions.** Instructions used in this study that have no direct effects when executed."  />
<p class="caption">
Figure 2.1: <strong>No-operation instructions.</strong> Instructions used in this study that have no direct effects when executed.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:Avida-instructions-Hardware-control-Table"></span>
<img src="images/Avida_instructions_Hardware_control_Table.png" alt="**Hardware control instructions.** Instructions used in this study that control the reading/writing of the Avidian genome and handle memory."  />
<p class="caption">
Figure 2.2: <strong>Hardware control instructions.</strong> Instructions used in this study that control the reading/writing of the Avidian genome and handle memory.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:Avida-instructions-Math-Table"></span>
<img src="images/Avida_instructions_Math_Table.png" alt="**Math instructions.** Instructions used in this study that execute mathematical operations."  />
<p class="caption">
Figure 2.3: <strong>Math instructions.</strong> Instructions used in this study that execute mathematical operations.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:Avida-instructions-Interaction-Table"></span>
<img src="images/Avida_instructions_Interaction_Table.png" alt="**Interaction instructions.** Instructions used in this study that enable communication and interaction among cells within a multicellular organism."  />
<p class="caption">
Figure 2.4: <strong>Interaction instructions.</strong> Instructions used in this study that enable communication and interaction among cells within a multicellular organism.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:Avida-instructions-Biological-Table"></span>
<img src="images/Avida_instructions_Biological_Table.png" alt="**Biological instructions.** Instructions used in this study that underpin cellular differentiation, multicellular development, and reproduction."  />
<p class="caption">
Figure 2.5: <strong>Biological instructions.</strong> Instructions used in this study that underpin cellular differentiation, multicellular development, and reproduction.
</p>
</div>
<p>The metabolism of digital cells involves the completion of logic functions, where the execution of a series of genomic instructions enables the cell to export the result of a logic operation on input bitstrings. The nine possible logic functions that can be performed are NOT, NAND, AND, ORNOT, OR, ANDNOT, NOR, XOR, and EQUALS. The execution of each function enables the cell to sequester resources from a virtual resource pool, where there is one resource pool per organism per function. Each resource pool has a constant in-flow rate of one unit per update (the standard unit of time in Avida), while at the same time 1% of the available resources flow out, limiting total accumulation to 100 units. When a cell exports the result of a function, it uptakes 5% of the available resource associated with that function. For each function, we also specify a per-site probability of mutation each time a function is performed, referred to a ``function mutagen level’’ (FML). Functions NOT and NAND are always non-mutagenic and thus have an FML of 0. For the treatments described in this work, the FML of the remaining seven functions increases with the complexity of the function (see Table <a href="materials-and-methods.html#fig:FML">2.6</a>. The resources acquired by successful execution of functions are required for an organism to reproduce.</p>
<div class="figure"><span style="display:block;" id="fig:FML"></span>
<img src="images/SI_Table_FML.png" alt="**Biological instructions.** Instructions used in this study that underpin cellular differentiation, multicellular development, and reproduction."  />
<p class="caption">
Figure 2.6: <strong>Biological instructions.</strong> Instructions used in this study that underpin cellular differentiation, multicellular development, and reproduction.
</p>
</div>
</div>
<div id="workload" class="section level3" number="2.1.3">
<h3><span class="header-section-number">2.1.3</span> Workload</h3>
<p>A cell’s workload is defined as:
<span class="math display">\[ w = \sum_{i \in F} N_i \frac{FML_i}{FML_{AND}} \]</span>
where <span class="math inline">\(F \equiv\)</span> {NOT, NAND, AND, ORNOT, OR, ANDNOT, NOR, XOR, EQUALS}, <span class="math inline">\(N_i\)</span> is the number of times function <span class="math inline">\(i\)</span> is performed, <span class="math inline">\(FML_i\)</span> is the mutagenic level of function <span class="math inline">\(i\)</span> (as specified in Table <a href="materials-and-methods.html#fig:FML">2.6</a>, and <span class="math inline">\(FML_{AND}\)</span> is the base <span class="math inline">\(FML\)</span> (i.e., the mutagenic level of the function AND).
We define propagule workload difference as the mean workload of propagule-ineligible cells minus the mean workload of propagule-eligible cells.</p>
</div>
<div id="mutations" class="section level3" number="2.1.4">
<h3><span class="header-section-number">2.1.4</span> Mutations</h3>
<p>Mutations to the genome, in which one instruction is replaced by another randomly selected instruction, occur during two different processes: organismal reproduction and the execution of ``dirty work’’ (logic tasks with FML <span class="math inline">\(&gt;\)</span> 0).
Mutations during reproduction only occur during propagule production (not tissue accretion) with probability 0.01 per site. On average this results in one new mutation per propagule produced. Dirty work mutations occur immediately after a cell performs a task associated with a strictly positive function mutagen level. Each site in the genome of the performing cell experiences mutations with probability given by the FML of the task executed (Table <a href="materials-and-methods.html#fig:FML">2.6</a>. More details behind the dirty work framework are provided in Goldsby et al. 2014 <span class="citation">(<a href="#ref-goldsby2014evolutionary" role="doc-biblioref">Goldsby et al. 2014</a>)</span>.</p>
</div>
<div id="baseline-evolution-experiment" class="section level3" number="2.1.5">
<h3><span class="header-section-number">2.1.5</span> Baseline evolution experiment</h3>
<p>The ancestral organism is unicellular with a genome of 100 instructions. Execution of the ancestral genome initially enables only a few simple behaviors. Specifically, the ancestor completes the task NOT a single time, subsequently rotates counterclockwise and then tries to produce a propagule cell. This reproductive process fails since it only has collected ~5 units of resource. It repeats this execution loop until it has enough resources to produce a propagule (taking roughly 1800 updates for the first generation). The population has a maximum size of 1000 organisms and evolves for a total of 1 million updates with a reproductive mutation probability of 0.01 and a dirty work mutation probability defined by whatever mutagenic functions evolve in the organisms.</p>
</div>
</div>
<div id="evolution-of-multicellularity" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Evolution of multicellularity</h2>
<p>Within our system, multicellularity is generally beneficial to organisms as a result of the increased ability of multiple cells to rapidly access resources that can be devoted to propagule production. Here we explore how the efficiency of resource consumption affects the evolution of multicellularity. Within our original experiments, each cell was able to consume 5% of the available resources each time a task was performed regardless of the number of cells in the organism. Here, we observe what happens when we increase the efficiency of resource uptake from 5% to 8%. This efficiency is applied at the cell level and thus is available to cells within either a unicellular or multicellular context. As can be seen in Figure <a href="materials-and-methods.html#fig:mc2">2.7</a>, increasing efficiency increases the percentage of replicates that evolve to be multicellular.</p>
<div class="figure"><span style="display:block;" id="fig:mc2"></span>
<img src="images/Figure_S1_Percent_multicellular_by_efficiency_level.png" alt="**Percent of evolutionary runs that evolve multicellularity as a function of resource acquisition efficiency.** For each treatment, depicted along the x-axis, we change the efficiency with which cells are able to acquire resources, and thus the benefits of multicellularity. Our original treatment (5%) has 1,000 replicates. The other treatments have 500 replicates. As the efficiency of resource consumption increases, so does the percentage of runs that evolve to be multicellular."  />
<p class="caption">
Figure 2.7: <strong>Percent of evolutionary runs that evolve multicellularity as a function of resource acquisition efficiency.</strong> For each treatment, depicted along the x-axis, we change the efficiency with which cells are able to acquire resources, and thus the benefits of multicellularity. Our original treatment (5%) has 1,000 replicates. The other treatments have 500 replicates. As the efficiency of resource consumption increases, so does the percentage of runs that evolve to be multicellular.
</p>
</div>
</div>
<div id="cost-of-multicellularity" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Cost of multicellularity</h2>
<p>We added an extrinsic cost of multicellularity for the purposes of testing how readily populations would revert to back to unicells. This cost was implemented as a time delay for producing a propagule from a multicellular organism. During this delay the organism and all its constituent cells are inert – they cannot perform tasks, acquire resources, reproduce, or communicate. Additionally, the organism remains at risk for being replaced by a reproducing competitor.</p>
<p>To confirm the costly nature of this time delay, we reran the evolution of our initially unicellular digital populations varying this delay from 0 updates (as in our original treatment) to 512 updates. We varied this multicellularity cost from 0 updates (as in our original evolution treatment) to 512 updates, incrementing by powers of 2 (e.g., 0, 1, 2, 4, 8, 16, etc.). As expected, fewer replicates evolved multicells as this cost was increased (Fig. <a href="materials-and-methods.html#fig:multi-cost">2.8</a>).</p>
<div class="figure"><span style="display:block;" id="fig:multi-cost"></span>
<img src="images/Percent_multi_by_cost.png" alt="**Percent of evolutionary runs that evolve multicellularity as a function of time delay cost.** For each treatment, depicted along the x-axis, we change the cost of propagule production, and thus the benefits of multicellularity. We ran 1000 replicates of each treatment. The percentage of runs that evolve multicellularity decreases as the time delay cost increases."  />
<p class="caption">
Figure 2.8: <strong>Percent of evolutionary runs that evolve multicellularity as a function of time delay cost.</strong> For each treatment, depicted along the x-axis, we change the cost of propagule production, and thus the benefits of multicellularity. We ran 1000 replicates of each treatment. The percentage of runs that evolve multicellularity decreases as the time delay cost increases.
</p>
</div>
</div>
<div id="entrenchment" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Entrenchment</h2>
<p>In studies of molecular evolution, a focal substitution is said to be entrenched by subsequent substitutions if it becomes relatively more deleterious to revert as a result of those subsequent substitutions <span class="citation">(<a href="#ref-shah2015contingency" role="doc-biblioref">Shah, McCandlish, and Plotkin 2015</a>; <a href="#ref-flynn2017inference" role="doc-biblioref">Flynn et al. 2017</a>; <a href="#ref-starr2018pervasive" role="doc-biblioref">Starr et al. 2018</a>; <a href="#ref-biswas2019epistasis" role="doc-biblioref">Biswas et al. 2019</a>)</span>. Here, because we are examining the entrenchment of a (multicellularity), we require an entrenchment metric that integrates across all possible mutations that can cause reversion to unicellularity. Specifically, we assess the degree to which multicellularity has become entrenched, by performing the following steps. First, we create a new population with 1000 copies of the organism when it was born. We then add an externally imposed cost of multicellularity. We run the population with this new multicellularity cost and observe if it reverts to unicellularity within 100 generations (assessed by measuring whether mean organism size has dropped below 2 cells after 100 generations of growth). We repeat this measurement with the following multicellularity cost values: 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048. To account for the stochastic nature of this assay, especially given the effect of dirty work on the organisms, we repeat the measurement three times for each organism, cost, and time point. We then compute a stability value for each genotype as the multicellular cost at which the probability of reversion to unicellularity <span class="math inline">\(= 0.5\)</span> using a logistic fit to the data (Fig. <a href="materials-and-methods.html#fig:dw-entrench-comparison">2.9</a>). Finally, entrenchment is computed as the difference in stability value at the final and transition time points.</p>
<div class="figure"><span style="display:block;" id="fig:dw-entrench-comparison"></span>
<img src="images/Figure_S3_Dirty_work_Entrenchment_20DEC22.png" alt="**A comparison of the stability of multicellularity at the transition and final time points of our &quot;Dirty Work&quot; evolution experiment.** The stability of multicellularity was measured for each of the 65 genotypes that evolved multicellularity at the transition (yellow) and final (green) time points of our experiment. The full range of cost values is shown with the outcomes of each replicate plotted as an indicator variable of the dominance of unicellular revertants. Logistic fits of the indicator variable are shown in yellow and green, representing the transition and final time points, respectively."  />
<p class="caption">
Figure 2.9: <strong>A comparison of the stability of multicellularity at the transition and final time points of our “Dirty Work” evolution experiment.</strong> The stability of multicellularity was measured for each of the 65 genotypes that evolved multicellularity at the transition (yellow) and final (green) time points of our experiment. The full range of cost values is shown with the outcomes of each replicate plotted as an indicator variable of the dominance of unicellular revertants. Logistic fits of the indicator variable are shown in yellow and green, representing the transition and final time points, respectively.
</p>
</div>
</div>
<div id="generation-and-assessment-of-unicellular-revertants" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Generation and assessment of unicellular revertants</h2>
<p>To understand the forces that govern whether or not a multicell is entrenched, we create all possible single-locus mutants at two time points: the “transitional” multicellular organism along the line of descent of the dominant multicell and “final” multicellular organism within this same lineage. For each single-locus mutant, we assess its viability as well as its fitness.</p>
<p>The first step in the assessment pipeline is the viability assay, which is a simple test to rule out organisms that are nonviable from further computationally expensive assessment. To assess a mutant’s viability, we run the mutated genome for an extended period of time (up to 10,000 updates) without mutations occurring as the result of dirty work. If, within this time period, the mutated organism is able to accumulate the resources required for reproduction and to execute the instruction to create a new organism, we consider it viable and stop the viability assay. Viable mutants are then further classified as either multicellular (indicating that it executed tissue accretion instructions resulting in cellular reproduction and thus contains more than two cells) or unicellular.</p>
<p>This viability assay is used as a gateway to further screening and thus its conclusions are refined by the subsequent growth assay. For example, since the organism is running its code in isolation it does not actually produce an offspring organism. Additionally, it may not contain propagule-eligible cells and thus may not be able to produce offspring when grown in a population under standard conditions. If so, later assays will classify it as nonviable. Similarly, the organism may be unicellular at the time of its first successful execution of the propagule production instruction and may later execute tissue accretion instructions, becoming multicellular. If so, later assays will classify it as multicellular.</p>
<div id="growth-assay-and-measuring-relative-fitness" class="section level3" number="2.5.1">
<h3><span class="header-section-number">2.5.1</span> Growth assay and measuring relative fitness</h3>
<p>To measure the fitness of a multicellular progenitor strain or a unicellular revertant strain, we create a solitary organism with the genome of interest and place it into an empty population with room for 62 organisms. The organism is allowed up to 50,000 updates to reach a population size threshold of 32 organisms. Once this threshold is reached, or exceeded (multiple simultaneous reproduction events can push populations above the 32 organism threshold), we calculate the average rate of increase, or Malthusian parameter, <span class="math inline">\(M\)</span>, as
<span class="math display">\[\begin{equation}
\ M = log(N_f / N_i )/T
\end{equation}\]</span>
where <span class="math inline">\(N_f\)</span> and <span class="math inline">\(N_i\)</span> are the final and initial number of organisms in the population and <span class="math inline">\(T\)</span> is the time it took the organism to fill the entire population. In the case that the organism fails to fill the entire population by the end of the growth assay, <span class="math inline">\(T\)</span> is set to the maximum allowed time of 50,000 updates.</p>
<p>To account for the effects of mutation, we repeat this process 100 times for each unicell revertant genotype and its multicell progenitor strain with different random number ``seeds,’’ leading to a different set of mutations and different cell behaviors. The fitness of a given unicell revertant relative to its multicell progenitor is defined as the ratio of their average Malthusian parameters across these replicate measurements.</p>
</div>
<div id="identification-of-unicellular-revertants" class="section level3" number="2.5.2">
<h3><span class="header-section-number">2.5.2</span> Identification of unicellular revertants</h3>
<p>Some mutations that occur over the course of the growth assay could result in data that may no longer be suitable for our analyses. For example, mutations that cause an organism to re-evolve multicellularity can occur during the growth assay, confounding our ability to accurately measure the relative fitness of revertants. Another potentially confounding scenario is that the organism we initiate the population with may fail to divide in the allotted time. This can occur if the initial organism flags itself as “propagule ineligible” or if it incurs a lethal mutation, as a result of dirty work, prior to the first propagule production event. To avoid these potentially confounding effects, we employ a two-step data filtering pipeline.</p>
<p>We begin by assessing whether the organisms in the population have remained unicellular for the entire duration of the growth assay. To do this, we check that the total number of cells in the population at the end of the growth assay is not greater than the total number of organisms. In total, ~42.5% of our growth rate assays are “contaminated” with re-evolved multicells. We see considerable diversity within our unicell revertant genotypes; a single unicell genotype can produce a variety of outcomes: some which remain unicellular and others which re-evolve multicellularity. This is fully in line with our expectation that different mutations will occur in organisms started with different random seeds. While some multicellular progenitor genotypes appear to produce revertants that re-evolve multicellularity at higher rates than others, we find that the probability distributions for re-evolving multicellularity are quite similar at the transition and final time points (Fig. <a href="materials-and-methods.html#fig:dw-filter-1-reevolving-multicellularity">2.10</a>).</p>
<div class="figure"><span style="display:block;" id="fig:dw-filter-1-reevolving-multicellularity"></span>
<img src="images/Figure_S4_Dirty_work_Populations_that_re-evolve_multicellularity_NEW_14MAR23.png" alt="**Filtering unicellular revertants from the &quot;Dirty Work&quot; experiment that re-evolve multicellularity.** The mean and standard error for the percent of replicate populations that remain unicellular for the transition (yellow) and final (green) time points of our experiment. Unicellular revertant genotypes are grouped according to their multicellular parental genotypes and ranked from least to most replicates that remain unicellluar for each time point independently. **Inset** Schematic representation of our filtering criterion using data aggregated across all genotypes and both time points. We filter out any population in which the average organism size exceeds 1, indicating that multicellularity has re-evolved. Grey color indicates replicates that fail to pass the filter, black indicates those that do."  />
<p class="caption">
Figure 2.10: <strong>Filtering unicellular revertants from the “Dirty Work” experiment that re-evolve multicellularity.</strong> The mean and standard error for the percent of replicate populations that remain unicellular for the transition (yellow) and final (green) time points of our experiment. Unicellular revertant genotypes are grouped according to their multicellular parental genotypes and ranked from least to most replicates that remain unicellluar for each time point independently. <strong>Inset</strong> Schematic representation of our filtering criterion using data aggregated across all genotypes and both time points. We filter out any population in which the average organism size exceeds 1, indicating that multicellularity has re-evolved. Grey color indicates replicates that fail to pass the filter, black indicates those that do.
</p>
</div>
<p>We next determine whether the organisms used to initiate the population have successfully reproduced at least once in the allotted 50,000 generations. To do this, we simply check that the total number of cells in the population is greater than 1. In total, 19% of our remaining replicate populations fail to successfully reproduce. As above, we find that some multicell progenitor genotypes are more likely than others to produce unicellular revertants that fail to successfully reproduce. However, in this case, we find that multicells from the final time point are more likely to produce inviable unicellular revertants than transition multicells (Fig. <a href="materials-and-methods.html#fig:dw-filter-2-failure-to-divide">2.11</a>).</p>
<div class="figure"><span style="display:block;" id="fig:dw-filter-2-failure-to-divide"></span>
<img src="images/Figure_S5_Dirty_work_Populations_that_fail_to_divide_NEW_14MAR23.png" alt="**Filtering unicellular revertants from the &quot;Dirty Work&quot; experiment that fail to reproduce.** The mean and standard error for the percent of replicate populations that successfully reproduce for the transition (yellow) and final (green) time points of our experiment. Unicellular revertant genotypes are grouped according to their multicellular parental genotypes and ranked from least to most replicates that remain unicellluar for each time point independently.  **Inset** Schematic representation of our filtering criterion using data aggregated across all genotypes and both time points. We filter out any population in which the number of organisms at the end of the growth assay is less than 2, indicating that the organism that seeded the population has failed to reproduce. Grey color indicates replicates that fail to pass the filter, black indicates those that do."  />
<p class="caption">
Figure 2.11: <strong>Filtering unicellular revertants from the “Dirty Work” experiment that fail to reproduce.</strong> The mean and standard error for the percent of replicate populations that successfully reproduce for the transition (yellow) and final (green) time points of our experiment. Unicellular revertant genotypes are grouped according to their multicellular parental genotypes and ranked from least to most replicates that remain unicellluar for each time point independently. <strong>Inset</strong> Schematic representation of our filtering criterion using data aggregated across all genotypes and both time points. We filter out any population in which the number of organisms at the end of the growth assay is less than 2, indicating that the organism that seeded the population has failed to reproduce. Grey color indicates replicates that fail to pass the filter, black indicates those that do.
</p>
</div>
</div>
</div>
<div id="distribution-of-fitness-effects-of-unicellular-reversion-mutations" class="section level2" number="2.6">
<h2><span class="header-section-number">2.6</span> Distribution of fitness effects of unicellular reversion mutations</h2>
<p>Using only the runs that passed both of our quality control filters (see text above), we then explored the distribution of fitness effects of reversion mutations for each multicellular lineage at the time of the transition to multicellularity and at the final time point of the evolutionary run (Fig. <a href="materials-and-methods.html#fig:dw-DFERMs">2.12</a>.</p>
<div class="figure"><span style="display:block;" id="fig:dw-DFERMs"></span>
<img src="images/Figure_S6_Relative_fitness_distributions_20DEC22.png" alt="**Distribution of fitness effects of unicellular reversion mutations from the &quot;Dirty Work&quot; experiment at the transition and final time points.** The empirical distribution of fitness effects of reversion mutations for each lineage that evolved multicellularity in our &quot;Dirty work&quot; experiment. Yellow indicates the distribution at the transition point, green at the final time point. Counts are proportional to the adjusted number of unicellular revertants."  />
<p class="caption">
Figure 2.12: <strong>Distribution of fitness effects of unicellular reversion mutations from the “Dirty Work” experiment at the transition and final time points.</strong> The empirical distribution of fitness effects of reversion mutations for each lineage that evolved multicellularity in our “Dirty work” experiment. Yellow indicates the distribution at the transition point, green at the final time point. Counts are proportional to the adjusted number of unicellular revertants.
</p>
</div>
</div>
<div id="reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations" class="section level2" number="2.7">
<h2><span class="header-section-number">2.7</span> Reversion mutation identities and the determinants of the number of viable reversion mutations</h2>
<p>To further understand the determinants of the number of viable reversion mutations, we explored the identity of the mutations that caused reversion from multicellularity to unicellularity at the transition and final time points of our experiment. For each time point we recorded the following: the identity of the mutant operation that caused reversion, the identity of the wild-type operation that was replaced, and the position in the genome. Figure <a href="materials-and-methods.html#fig:case-study-reversion-mutation-identities">2.13</a> gives a representative example reversion mutation network for our case study lineage at the final time point which highlights 3 major categories of mutation that lead to a reversion to unicellularity. First, the <span class="math inline">\(&lt;h\_divide\_remote&gt;\)</span> operation, which encodes propagule production, can be inserted at several sites in the genome (usually upstream of the operation encoding tissue accretion) to repress tissue accretion in a dominant epistatic manner. Second, the regulatory loci controlling the expression of tissue accretion can be disrupted (e.g., the loss of the <span class="math inline">\(&lt;if\_res\_less\_than\_thresh&gt;\)</span> operation at position 12 in our final case study genome). Third, loss-of-function mutations occur when the tissue accretion operation, <span class="math inline">\(&lt;h\_divide\_local&gt;\)</span>, is substituted for another operation that does not disrupt propagule production.</p>
<p>When we look at the spectrum of reversion mutations across all 65 lineages that evolved multicellularity, we find that the same basic pattern emerges at both the transition and final time points (Figs. <a href="materials-and-methods.html#fig:reversion-mutation-identities">2.14</a>-<a href="materials-and-methods.html#fig:wt-operation-identities">2.15</a>). Loss-of-function mutations are the most common cause of reversion followed closely by dominant epistatic mutations. Changes in regulatory elements make up the third most common category of reversion mutation and occur at a slightly higher frequency in the final genomes than the transition genomes. The biggest difference in mutation spectrum between the transition and final time points is in the frequency at which <span class="math inline">\(&lt;nop\_x&gt;\)</span> operations are replaced (Fig. <a href="materials-and-methods.html#fig:wt-operation-identities">2.15</a>). However, this difference is caused by the fact that the ancestral unicell that was used to initiate each evolutionary run had an extremely minimal genome composed largely of <span class="math inline">\(&lt;nop\_x&gt;\)</span> operations (a null operation). These large blocks of null operations are gradually replaced over time as the population evolves; thus, earlier time points will contain more null operations than later ones. When adjusted for the frequency of the <span class="math inline">\(&lt;nop\_x&gt;\)</span> operation in the multicell genomes (<span class="math inline">\(24.4\%\)</span> at the time of transition vs. <span class="math inline">\(1.9\%\)</span> at the final time point), the difference in frequency effectively goes to zero.</p>
<p>Another factor we hypothesized could lead to a decrease in the number of viable reversion mutations was the presence of redundant tissue accretion mutations. Since the primary routes to achieving unicellularity involve replacing or silencing the expression of the tissue accretion instruction, it is possible that multicells could become more robust to reversion by accumulating secondary functional copies of the <span class="math inline">\(&lt;h\_divide\_local&gt;\)</span> operation. We found some evidence of a trend in this direction (Fig. <a href="materials-and-methods.html#fig:redundant-tissue-accretion-instructions">2.16</a>).</p>
<p></p>
<div class="figure"><span style="display:block;" id="fig:case-study-reversion-mutation-identities"></span>
<img src="images/Final_case_study_mutation_network_30DEC22.png" alt="**Identities of the mutations that caused reversion to unicellularity in our case study lineage at the final time point.** Here we show how the reversion mutations map onto the positions in the genome that were mutated and the operations they replaced for our case study. The color of each node corresponds to the operation and genome position is labeled for each operation that was replaced in the final multicell genome."  />
<p class="caption">
Figure 2.13: <strong>Identities of the mutations that caused reversion to unicellularity in our case study lineage at the final time point.</strong> Here we show how the reversion mutations map onto the positions in the genome that were mutated and the operations they replaced for our case study. The color of each node corresponds to the operation and genome position is labeled for each operation that was replaced in the final multicell genome.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:reversion-mutation-identities"></span>
<img src="images/Mutant_operations_at_transition_and_final_time_point_20DEC22.png" alt="**Changes in reversion mutation identities from the transition to final time point.** The number of times different mutant operations caused reversion to unicellularity."  />
<p class="caption">
Figure 2.14: <strong>Changes in reversion mutation identities from the transition to final time point.</strong> The number of times different mutant operations caused reversion to unicellularity.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:wt-operation-identities"></span>
<img src="images/Wildtype_operations_at_transition_and_final_time_point_20DEC22.png" alt="**Changes in replaced wild-type operation identities from the transition to final time point.** The number of times that replacement of different wild-type operations caused reversion to unicellularity."  />
<p class="caption">
Figure 2.15: <strong>Changes in replaced wild-type operation identities from the transition to final time point.</strong> The number of times that replacement of different wild-type operations caused reversion to unicellularity.
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:redundant-tissue-accretion-instructions"></span>
<img src="images/Redundant_tissue_accretion_operations_23DEC22.png" alt="**Effect of redundant tissue accretion instructions.** To investigate the possibility that redundant tissue accretion instruction operations could reduce the number of viable reversion mutations, we examined the genomes of multicells at the transition (yellow) and final (green) time point. (**A**) A comparison of the number of tissue accretion instructions found in each genome. (**B**) The effect of tissue accretion instruction copy number on the number of viable unicellular revertants."  />
<p class="caption">
Figure 2.16: <strong>Effect of redundant tissue accretion instructions.</strong> To investigate the possibility that redundant tissue accretion instruction operations could reduce the number of viable reversion mutations, we examined the genomes of multicells at the transition (yellow) and final (green) time point. (<strong>A</strong>) A comparison of the number of tissue accretion instructions found in each genome. (<strong>B</strong>) The effect of tissue accretion instruction copy number on the number of viable unicellular revertants.
</p>
</div>
</div>
<div id="fidelity" class="section level2" number="2.8">
<h2><span class="header-section-number">2.8</span> Fidelity</h2>
<p>We directly measure the per site mutation rate (<span class="math inline">\(\mu\)</span>) to assess the fidelity of replication in both multicellular and unicellular genotypes. To do this, we record parent and offspring propagule genomes for each replication event that occurs during our growth assays. Each pair of parent and offspring genomes is then aligned using the aphid R package <span class="citation">(<a href="#ref-aphid" role="doc-biblioref">Wilkinson 2019</a>)</span> and the instruction equivalent of single nucleotide polymorphisms (SNPs) are called wherever there is a mismatch. To restrict our analyses to mutations that could have been the result of either dirty work or the basal mutation rate we record only these SNPs (insertions and deletions are ignored, for example, as they are the result of errors in the copy loop and not from the effects of dirty work). Finally, duplicate mutation calls are removed by comparing SNPs found in offspring derived from the same parent. We repeat this process 100 times for each genotype. We calculate <span class="math inline">\(\mu\)</span> as the total number of unique mutations divided by the number of aligned positions from all offspring genomes. Fidelity is reported as one minus the per site mutation rate (i.e., <span class="math inline">\(1 - \mu\)</span>).</p>
<div class="figure"><span style="display:block;" id="fig:mutation-rates"></span>
<img src="images/Mutation_rate_by_cell_differentation_status_27JULY20.png" alt="**Mutation rate by cell differentiation status.**"  />
<p class="caption">
Figure 2.17: <strong>Mutation rate by cell differentiation status.</strong>
</p>
</div>
<div class="figure"><span style="display:block;" id="fig:case-study-over-time-mutation-rates"></span>
<img src="images/Workload_by_fidelity_case_study_3416_30JULY20.png" alt="**Evolutionary trajectory of a case study population (3416) in workload and fidelity space.**"  />
<p class="caption">
Figure 2.18: <strong>Evolutionary trajectory of a case study population (3416) in workload and fidelity space.</strong>
</p>
</div>
</div>
<div id="effect-of-time-spent-evolving-as-a-multicell" class="section level2" number="2.9">
<h2><span class="header-section-number">2.9</span> Effect of time spent evolving as a multicell</h2>
<p>In the main text, we treated all lineages that had evolved multicellularity by the end of their evolutionary run equally. However, we observed considerable variation in the time at which multicellularity evolved and this could lead to differences in entrenchment because genomes would spend different periods of time evolving in a multicellular context <span class="citation">(<a href="#ref-libby2016stabilizing" role="doc-biblioref">Libby et al. 2016</a>)</span>. To test this prediction, we examined how several key parameters in our study (entrenchment, number of unicellular revertants, revertant relative fitness, revertant growth rate, and multicell growth rate) changed as a function of the number of generations a population spent evolving as a multicell. Consistent with predictions, entrenchment increased significantly with greater time spent evolving as a multicell (Fig. <a href="materials-and-methods.html#fig:dw-time-as-mc">2.19</a>A; Spearman’s rank-order correlation, <span class="math inline">\(\rho=0.3\)</span>, <span class="math inline">\(p=0.02\)</span>). Only the change in the number of unicellular revertants did not show a significant correlation with time spent evolving as a multicell (Fig. <a href="materials-and-methods.html#fig:dw-time-as-mc">2.19</a>B; Spearman’s rank-order correlation, <span class="math inline">\(\rho=0.08\)</span>, <span class="math inline">\(p=0.53\)</span>). The change in unicell revertant relative fitness declined strongly as a function of time spent evolving as a multicell (Fig. <a href="materials-and-methods.html#fig:dw-time-as-mc">2.19</a>C; Spearman’s rank-order correlation, <span class="math inline">\(\rho=-0.34\)</span>, <span class="math inline">\(p=5.5e-3\)</span>) despite the positive correlation between revertant growth rate and time as multicell (Fig. <a href="materials-and-methods.html#fig:dw-time-as-mc">2.19</a>D; Spearman’s rank-order correlation, <span class="math inline">\(\rho=0.34\)</span>, <span class="math inline">\(p=6.1e-3\)</span>) because the growth rate improvements were vastly outstripped by those of their multicellular progenitor strains (Fig. <a href="materials-and-methods.html#fig:dw-time-as-mc">2.19</a>E; Spearman’s rank-order correlation, <span class="math inline">\(\rho=0.86\)</span>, <span class="math inline">\(p&lt;2.2e-16\)</span>).</p>
<div class="figure"><span style="display:block;" id="fig:dw-time-as-mc"></span>
<img src="images/Figure_S8_Time_as_multicell_10NOV22.png" alt="**Effect of time spent evolving as a multicell.** We examine the correlation between time spent evolving as a multicell and the change in five key parameters in our experiment: entrenchment (**A**), number of unicellular revertants (**B**), revertant relative fitness (**C**), revertant growth rate (**D**), and multicell growth rate (**E**). Lineages that evolved germ-soma differentiation appear in red, undifferentiated lineages in blue."  />
<p class="caption">
Figure 2.19: <strong>Effect of time spent evolving as a multicell.</strong> We examine the correlation between time spent evolving as a multicell and the change in five key parameters in our experiment: entrenchment (<strong>A</strong>), number of unicellular revertants (<strong>B</strong>), revertant relative fitness (<strong>C</strong>), revertant growth rate (<strong>D</strong>), and multicell growth rate (<strong>E</strong>). Lineages that evolved germ-soma differentiation appear in red, undifferentiated lineages in blue.
</p>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-biswas2019epistasis" class="csl-entry">
Biswas, Avik, Allan Haldane, Eddy Arnold, and Ronald M Levy. 2019. <span>“Epistasis and Entrenchment of Drug Resistance in HIV-1 Subtype b.”</span> <em>eLife</em> 8: e50524.
</div>
<div id="ref-Chow2004" class="csl-entry">
Chow, S., C. O. Wilke, C. Ofria, R. E. Lenski, and C. Adami. 2004. <span>“Adaptive Radiation from Resource Competition in Digital Organisms.”</span> <em>Science</em> 305: 84–86.
</div>
<div id="ref-flynn2017inference" class="csl-entry">
Flynn, William F, Allan Haldane, Bruce E Torbett, and Ronald M Levy. 2017. <span>“Inference of Epistatic Effects Leading to Entrenchment and Drug Resistance in HIV-1 Protease.”</span> <em>Molecular Biology and Evolution</em> 34 (6): 1291–1306.
</div>
<div id="ref-goldsby2012task" class="csl-entry">
Goldsby, Heather J, Anna Dornhaus, Benjamin Kerr, and Charles Ofria. 2012. <span>“Task-Switching Costs Promote the Evolution of Division of Labor and Shifts in Individuality.”</span> <em>Proceedings of the National Academy of Sciences</em> 109 (34): 13686–91.
</div>
<div id="ref-goldsby2014evolutionary" class="csl-entry">
Goldsby, Heather J, David B Knoester, Charles Ofria, and Benjamin Kerr. 2014. <span>“The Evolutionary Origin of Somatic Cells Under the Dirty Work Hypothesis.”</span> <em>PLoS Biology</em> 12 (5): e1001858.
</div>
<div id="ref-Lenski2003" class="csl-entry">
Lenski, R. E., C. Ofria, R. T. Pennock, and C. Adami. 2003. <span>“The Evolutionary Origin of Complex Features.”</span> <em>Nature</em> 423: 139–44.
</div>
<div id="ref-libby2016stabilizing" class="csl-entry">
Libby, Eric, Peter L Conlin, Ben Kerr, and William C Ratcliff. 2016. <span>“Stabilizing Multicellularity Through Ratcheting.”</span> <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em> 371 (1701): 20150444.
</div>
<div id="ref-ofria2004avida" class="csl-entry">
Ofria, Charles, and Claus O Wilke. 2004. <span>“Avida: A Software Platform for Research in Computational Evolutionary Biology.”</span> <em>Artificial Life</em> 10 (2): 191–229.
</div>
<div id="ref-shah2015contingency" class="csl-entry">
Shah, Premal, David M McCandlish, and Joshua B Plotkin. 2015. <span>“Contingency and Entrenchment in Protein Evolution Under Purifying Selection.”</span> <em>Proceedings of the National Academy of Sciences</em> 112 (25): E3226–35.
</div>
<div id="ref-starr2018pervasive" class="csl-entry">
Starr, Tyler N, Julia M Flynn, Parul Mishra, Daniel NA Bolon, and Joseph W Thornton. 2018. <span>“Pervasive Contingency and Entrenchment in a Billion Years of Hsp90 Evolution.”</span> <em>Proceedings of the National Academy of Sciences</em> 115 (17): 4453–58.
</div>
<div id="ref-aphid" class="csl-entry">
Wilkinson, Shaun. 2019. <span>“Aphid: An r Package for Analysis with Profile Hidden Markov Models.”</span> <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btz159">https://doi.org/10.1093/bioinformatics/btz159</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="distributed-dirt-model.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/kgskocelas/mt2index.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["supplemental_material.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
