<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Section 9 Fidelity | Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity</title>
  <meta name="description" content="This is the supplementary material accompanying the paper “Division of Labor Promotes the Entrenchment of Multicellularity”." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Section 9 Fidelity | Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is the supplementary material accompanying the paper “Division of Labor Promotes the Entrenchment of Multicellularity”." />
  <meta name="github-repo" content="kgskocelas/mt2" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Section 9 Fidelity | Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity" />
  
  <meta name="twitter:description" content="This is the supplementary material accompanying the paper “Division of Labor Promotes the Entrenchment of Multicellularity”." />
  

<meta name="author" content="Peter L. Conlin, Heather J. Goldsby, Eric Libby, Katherine G. Skocelas, William C. Ratcliff, Charles Ofria, Benjamin Kerr" />


<meta name="date" content="2023-08-21" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations.html"/>
<link rel="next" href="effect-of-time-spent-evolving-as-a-multicell.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet" />
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity</a></li>

<li class="divider"></li>
<li><a href="sankey-network-graphics.html#sankey-network-graphics" id="toc-sankey-network-graphics"><span class="toc-section-number">1</span> Sankey Network Graphics</a>
<ul>
<li><a href="sankey-network-graphics.html#transition-updated" id="toc-transition-updated"><span class="toc-section-number">1.1</span> Transition Updated</a></li>
<li><a href="sankey-network-graphics.html#final-updated" id="toc-final-updated"><span class="toc-section-number">1.2</span> Final Updated</a></li>
</ul></li>
<li><a href="avida.html#avida" id="toc-avida"><span class="toc-section-number">2</span> Avida</a>
<ul>
<li><a href="avida.html#time" id="toc-time"><span class="toc-section-number">2.1</span> Time</a></li>
<li><a href="avida.html#instructions-and-functions" id="toc-instructions-and-functions"><span class="toc-section-number">2.2</span> Instructions and functions</a></li>
<li><a href="avida.html#workload" id="toc-workload"><span class="toc-section-number">2.3</span> Workload</a></li>
<li><a href="avida.html#mutations" id="toc-mutations"><span class="toc-section-number">2.4</span> Mutations</a></li>
<li><a href="avida.html#baseline-evolution-experiment" id="toc-baseline-evolution-experiment"><span class="toc-section-number">2.5</span> Baseline evolution experiment</a></li>
</ul></li>
<li><a href="evolution-of-multicellularity.html#evolution-of-multicellularity" id="toc-evolution-of-multicellularity"><span class="toc-section-number">3</span> Evolution of multicellularity</a></li>
<li><a href="cost-of-multicellularity.html#cost-of-multicellularity" id="toc-cost-of-multicellularity"><span class="toc-section-number">4</span> Cost of multicellularity</a></li>
<li><a href="entrenchment.html#entrenchment" id="toc-entrenchment"><span class="toc-section-number">5</span> Entrenchment</a></li>
<li><a href="generation-and-assessment-of-unicellular-revertants.html#generation-and-assessment-of-unicellular-revertants" id="toc-generation-and-assessment-of-unicellular-revertants"><span class="toc-section-number">6</span> Generation and assessment of unicellular revertants</a>
<ul>
<li><a href="generation-and-assessment-of-unicellular-revertants.html#growth-assay-and-measuring-relative-fitness" id="toc-growth-assay-and-measuring-relative-fitness"><span class="toc-section-number">6.1</span> Growth assay and measuring relative fitness</a></li>
<li><a href="generation-and-assessment-of-unicellular-revertants.html#identification-of-unicellular-revertants" id="toc-identification-of-unicellular-revertants"><span class="toc-section-number">6.2</span> Identification of unicellular revertants</a></li>
</ul></li>
<li><a href="distribution-of-fitness-effects-of-unicellular-reversion-mutations.html#distribution-of-fitness-effects-of-unicellular-reversion-mutations" id="toc-distribution-of-fitness-effects-of-unicellular-reversion-mutations"><span class="toc-section-number">7</span> Distribution of fitness effects of unicellular reversion mutations</a></li>
<li><a href="reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations.html#reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations" id="toc-reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations"><span class="toc-section-number">8</span> Reversion mutation identities and the determinants of the number of viable reversion mutations</a></li>
<li><a href="fidelity.html#fidelity" id="toc-fidelity"><span class="toc-section-number">9</span> Fidelity</a></li>
<li><a href="effect-of-time-spent-evolving-as-a-multicell.html#effect-of-time-spent-evolving-as-a-multicell" id="toc-effect-of-time-spent-evolving-as-a-multicell"><span class="toc-section-number">10</span> Effect of time spent evolving as a multicell</a></li>
<li><a href="distributed-dirt-model.html#distributed-dirt-model" id="toc-distributed-dirt-model"><span class="toc-section-number">11</span> Distributed Dirt Model</a></li>
<li><a href="references.html#references" id="toc-references">References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supplementary Material for Division of Labor Promotes the Entrenchment of Multicellularity</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="fidelity" class="section level1" number="9">
<h1><span class="header-section-number">Section 9</span> Fidelity</h1>
<p>We directly measure the per site mutation rate (<span class="math inline">\(\mu\)</span>) to assess the fidelity of replication in both multicellular and unicellular genotypes. To do this, we record parent and offspring propagule genomes for each replication event that occurs during our growth assays. Each pair of parent and offspring genomes is then aligned using the aphid R package <span class="citation">(<a href="#ref-aphid" role="doc-biblioref">Wilkinson 2019</a>)</span> and the instruction equivalent of single nucleotide polymorphisms (SNPs) are called wherever there is a mismatch. To restrict our analyses to mutations that could have been the result of either dirty work or the basal mutation rate we record only these SNPs (insertions and deletions are ignored, for example, as they are the result of errors in the copy loop and not from the effects of dirty work). Finally, duplicate mutation calls are removed by comparing SNPs found in offspring derived from the same parent. We repeat this process 100 times for each genotype. We calculate <span class="math inline">\(\mu\)</span> as the total number of unique mutations divided by the number of aligned positions from all offspring genomes. Fidelity is reported as one minus the per site mutation rate (i.e., <span class="math inline">\(1 - \mu\)</span>).</p>
<!-- ```{r, mutation-rates, echo=FALSE, fig.pos='H', fig.cap="**Mutation rate by cell differentiation status.**"} -->
<!-- knitr::include_graphics("images/Mutation_rate_by_cell_differentation_status_27JULY20.png") -->
<!-- ``` -->
<!-- ```{r, case-study-over-time-mutation-rates, echo=FALSE, fig.pos='H', fig.cap="**Evolutionary trajectory of a case study population (3416) in workload and fidelity space.**"} -->
<!-- knitr::include_graphics("images/Workload_by_fidelity_case_study_3416_30JULY20.png") -->
<!-- ``` -->
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-aphid" class="csl-entry">
Wilkinson, Shaun. 2019. <span>“Aphid: An r Package for Analysis with Profile Hidden Markov Models.”</span> <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btz159">https://doi.org/10.1093/bioinformatics/btz159</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="reversion-mutation-identities-and-the-determinants-of-the-number-of-viable-reversion-mutations.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="effect-of-time-spent-evolving-as-a-multicell.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/kgskocelas/mt2%s",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": false,
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
